src/coreComponents/constitutiveDrivers/fluid/multiFluid/PVTDriverRunTest.hpp

```diff
@@ -80,7 +81,8 @@ void PVTDriver::runTest( FLUID_TYPE & fluid, arrayView2d< real64 > const & table
     integer const PHASE_FRACTION = outputCompressibility ? TEMP + 3 : TEMP + 2;
     integer const PHASE_DENSITY = PHASE_FRACTION + numPhases;
     integer const PHASE_VISCOSITY = outputMassDensity ? PHASE_DENSITY + 2*numPhases : PHASE_DENSITY + numPhases;
-    integer const PHASE_COMP = PHASE_VISCOSITY + numPhases;
+    integer const PHASE_ENTHALPY = isThermal ? PHASE_VISCOSITY + numPhases : PHASE_VISCOSITY;
+    integer const PHASE_COMP = PHASE_ENTHALPY + numPhases;
```

Comment:
sorry, this part is a bit hard to follow, but isn't that ..?
```suggestion
    integer const PHASE_ENTHALPY = PHASE_VISCOSITY + numPhases;
    integer const PHASE_COMP = isThermal ? PHASE_ENTHALPY + numPhases : PHASE_ENTHALPY;
```

On a side note, maybe this kind of formulation may be clearer, if you prefer:
```
    integer const PHASE_Y = PHASE_X + ( outputX ? numPhases : 0 );
```

Reply:
I see your point. Makes sense. I will add that.